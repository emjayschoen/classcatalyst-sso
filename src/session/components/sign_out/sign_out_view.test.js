import React from 'react';
import { shallow } from 'enzyme/build';
import SignOutView from './sign_out_view';

describe('<SignOutView />', () => {
  let subject;

  describe('when signed in', () => {
    beforeEach(() => {
      subject = shallow(<SignOutView signedIn signOut={jest.fn()} />);
    });

    it('renders correctly', () => {
      expect(subject).toMatchSnapshot();
    });
  });
  describe('when signed out', () => {
    beforeEach(() => {
      subject = shallow(<SignOutView signedIn={false} signOut={jest.fn()} />);
    });

    it('renders correctly', () => {
      expect(subject).toMatchSnapshot();
    });
  });
});
