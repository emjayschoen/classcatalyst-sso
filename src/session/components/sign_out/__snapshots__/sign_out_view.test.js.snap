// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignOutView /> when signed in renders correctly 1`] = `<Spinner />`;

exports[`<SignOutView /> when signed out renders correctly 1`] = `
<Redirect
  to="/"
/>
`;
